.github/workflows/publish.yml

```diff
@@ -85,6 +111,23 @@ jobs:
           chartpress --no-build
           ./tools/set-chart-yaml-annotations.py
 
+      - name: Publish images and chart with chartpress
+        if: steps.publishing.outputs.publishing
+        env:
+          GITHUB_REPOSITORY: "${{ github.repository }}"
+        run: |
           # Package the Helm chart and publish it to the gh-pages branch of
           # the jupyterhub/helm-chart repo.
           ./ci/publish
+
+      - name: Package helm chart as a CI artifact
+        if: steps.publishing.outputs.publishing == ''
+        run: helm package jupyterhub
+
+      # ref: https://github.com/actions/upload-artifact
+      - uses: actions/upload-artifact@v2
+        if: steps.publishing.outputs.publishing == ''
```

Comment:
Weird possible bug in actions: cannot use simpler `if: !steps.publishing.outputs.publishing` which produces a syntax error, when I'm pretty sure it should be fine.